sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -75,11 +104,32 @@ describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
       // Student created with expected first name, last name and more importantly the updated date
       // to fall between the most recent bridge file date and the mocked current date.
       const student = await createStudentWithExpectedData(simsDataUpdatedDate);
+
       // Student has submitted an application.
-      await saveFakeApplicationDisbursements(
+      const application = await saveFakeApplicationDisbursements(
         db.dataSource,
         { student },
-        { offeringIntensity: OfferingIntensity.partTime },
+        {
+          offeringIntensity: OfferingIntensity.partTime,
+        },
+      );
+      application.updatedAt = simsDataUpdatedDate;
```

Comment:
Please a generic comment after line 97 to explain the reason why we set the updatedAt to a particular date.